10_temperature_object/temperature.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+class Temperature
+  class << self
+    def from_fahrenheit(temp)
+      Temperature.new(fahrenheit: temp)
+    end
+
+    def from_celsius(temp)
+      Temperature.new(celsius: temp)
+    end
+  end
+
+  def initialize(options = {})
+    @fahrenheit = options[:fahrenheit]
```

Comment:
`options[:fahrenheit]` => that's okay, but a little bit old-fashioned way
use named variables, like
```ruby
def initialize(fahrenheit: nil, celsium: nil)
```